set.mm

```diff
@@ -749468,4 +749469,325 @@ to Davis and Putnam (1960).  (Contributed by David A. Wheeler,
 
 $( (End of Larry Lesyna's mathbox.) $)
 
+$(
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+  Mathbox for Adhemar
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+$)
+
+
+  $( Replacement of a nested antecedent with an outer antecedent.  Commuted
+     simplificated form of elimination of a nested antecedent.  Also holds
+     intuitionistically.  Polish prefix notation:  CCCpqrCsCqr .  (Contributed
```

Comment:
**Welcome to Metamath!**

I've launched `rewrap` and `regen discouraged` on this PR to help it pass our CI, I hope you don't mind.

For the "Polish prefix notation" lines to stand alone on a paragraph, you have to put one empty line (two newlines) before.